src/components/panel/panel.spec.js

```diff
@@ -542,6 +542,20 @@ describe('$mdPanel', function() {
       expect(PANEL_EL).not.toExist();
     });
 
+    it('should close when clickOutsideToClose set to true and ' +
+        'propagateContainerEvents is also set to true', function() {
+          var config = {
+            propagateContainerEvents: true,
+            clickOutsideToClose: true
+          };
+
+          openPanel(config);
```

Comment:
It might make the test slightly more robust if you had a check that the `PANEL_EL` exists, right after opening it. This should avoid cases where the test will pass, if the `openPanel` function didn't open it up for some reason.